config.go

```diff
@@ -422,21 +497,6 @@ func ParseConfig(fn string) (Configuration, error) {
 	}
 
 	// TLS configuration
-	clientKeyFilename, ok := input.Get(outType, "client_key")
```

Comment:
any idea why we removed this?

Reply:
I had to pull it because it was recreated up above and then swapped in for the rabbit variables that don't exist anymore.

Reply:
I originally implemented this then the CB team pulled it in but changed the var names.